### PR TITLE
add developing accessible products page

### PR DESCRIPTION
**Pull request by hollsk**
# What fresh hell is this?

It's a concise guide to building accessible products! 

## But you've already done this in 3 separate pages in this directory

I know, but this time it's _better_!

## Have you finally lost it or what

Obviously yes, but there is a method to my madness. 

The trigger for building this particular page is that we (or rather other people working for the company) sometimes commission work from 3rd parties, or from departments and groups who may have limited experience in actually building accessible content. 

Because we purposefully hire people who get what accessibility is, why it's important, and how it works, the rest of the Playbook assumes a certain level of knowledge and understanding. But if we're commissioning something from another company or even another part of the business where there are no FEDs, we can't rely on that assumption. 

So this particular page is for:

* 3rd party developers, where it will form part of the specification for their deliverables
* Internal developers who have no FEDs or connections to our department and who might be encountering accessibility for the very first time
* Non-technical people involved in procurement decisions in our company and in those from whom we are purchasing services (so they know what we're going to look for before we approve something)

### Yeah great but you've just repeated a load of stuff that's already in the other pages

This is true, and I (or somebody else, maybe even you, if you're volunteering) will fix that in another PR. We need to rejig these docs anyway as part of the "make the Playbook more of a guided experience" work. 

In the meantime, this page has a very particular use case that will hopefully save a bit of back-and-forth between FED Enablement, various commissioned developers, and assorted commissioning parties.

### CODE REVIEW

File: accessibility/developing-accessible-products.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Developing accessible products
+
+Businesses with an online presence must ensure they make their offering accessible to disabled users. This is a legal obligation that we are required to meet in multiple countries and jurisdictions. 
+
+Certain organisations such as scientific or educational institutions may be prevented by federal regulations from procuring products and services that do not meet accessibility requirements. 
+
+Commercial products are generally covered under civil rights legislation such as the Americans with Disabilities Act in the US or the Equality Act in the UK. If a commercial website is found to be accessible, the website owner can be sued for discrimination under various civil rights legislatory mechanisms.
```
Comment (jon-stevens) [inline accessibility/developing-accessible-products.md]: inaccessible?

Comment (hollsk) [reply accessibility/developing-accessible-products.md]: lol

Comment (jon-stevens) [inline accessibility/developing-accessible-products.md]:
```suggestion
Commercial products are generally covered under civil rights legislation such as the Americans with Disabilities Act in the US or the Equality Act in the UK. If a commercial website is found to be inaccessible, the website owner can be sued for discrimination under various civil rights legislatory mechanisms.
```

**Comment by jon-stevens** [comment]
LGTM 
image

**Comment by jpw** [comment]
I LIKE IT

File: accessibility/developing-accessible-products.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+The Fast Pass functionality is based on Axe, but bear in mind that the bundled Axe version may not be in sync with the official package, and you may get inconsistent results if you rely on the version bundled by Microsoft. 
+
+We expect your product to be free of errors that can be identified using this tool. 
+
+
+#### Testing with a screen reader
+
+The Accessibility Insights tool doesn’t cover screen reader testing. We do expect products to be usable in screen readers, so you should conduct your own testing with at least one of these. 
+
+You will need to learn how to use a screen reader, but they are not complicated tools. They are consumer software products, designed to allow ordinary users to interact with websites without a visual display.
```
[the remark this answers is not in the document]
Comment (hollsk) [reply accessibility/developing-accessible-products.md]: Yeah, but you're a thousand years old and can barely use a computer. 

(You also didn't have the handy guides I linked to. But mostly you're old and can barely use a computer)

**Comment by huckocs** [comment]
Is this something that could go on the accessibility site?

File: accessibility/developing-accessible-products.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (dotcode) [reply accessibility/developing-accessible-products.md]: All true, I admit.

**Comment by jpw** [comment]
@huckocs yes. Hollie and I had a chat, and I think it's probably best the playbook is the Source of Authority for this doc.

It's _possible_ we could do something clever to get this content in the a11y site (as per the internal design system site), but I think we can live with a bit of copy & paste for now until the a11y site is a bit more fleshed-out? I can't imagine this document changing frequently... whaddaya reckon?

**Comment by huckocs** [comment]
@jpw Yeah for sure. I was thinking more the content than it magically appearing there. Copy and paste is fine.

File: accessibility/developing-accessible-products.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (morgaan) [reply accessibility/developing-accessible-products.md]: Accessibility covers aging population thank god ;)

**Pull request merged**